manifests/mod/passenger.pp

```diff
@@ -546,6 +550,11 @@
         fail("Passenger config option :: passenger_load_shell_envvars is not introduced until version 4.0.20 :: ${passenger_installed_version} is the version reported")
       }
     }
+    if $passenger_preload_bundler {
+      if (versioncmp($passenger_preload_bundler, '6.0.13') < 0) {
```

Comment:
Shouldn't this line be `passenger_installed_version` not `passenger_preload_bundler`?

Reply:
Meh… completion operator error :facepalm: 

Thanks, I amended the commit to fix this!